### DISCUSSION TITLE
After opening the modal window the components disappeared

### DISCUSSION THREAD

**Discussion by DeNIKE666** [livewire/livewire#4449] [1. Help]
Hello, friends, help me solve the problem, I don't know what this is connected with, I have a list of users that I want to edit through a modal window, after clicking on the desired user, a modal window opens, and disappears (input field and selection)

**master layout**

`<script>
    window.addEventListener('edit-user', event => {
        $('#show-user').show();
    })
</script>
`

![Снимок экрана 2021-12-24 122025](https://user-images.githubusercontent.com/30469085/147328774-4386ad05-a671-4232-b329-7fb319f68ead.png)

**EditUser component**

![image](https://user-images.githubusercontent.com/30469085/147328863-fbc7fcb0-77f6-4af5-a12d-2b226ef79cef.png)

Everything renders well in the layout

![image](https://user-images.githubusercontent.com/30469085/147328982-3cc8f045-17b5-473c-b931-846ae700cdfe.png)

The selection disappears when the button is pressed.

![image](https://user-images.githubusercontent.com/30469085/147329097-3638bc33-eb11-4b01-991f-1ef5a45d446e.png)

**Comment by joshhanley** [comment]
@DeNIKE666 that looks like a morphdom issue. Can you share your blade view for you modal?
